docs/docs/about/contributing.md

```diff
@@ -78,18 +78,21 @@ Some notes on developing in Dagster:
 For development, run an instance of the webserver providing GraphQL service on a different port than the webapp, with any pipeline. For example:
 
 ```bash
-cd dagster/examples/docs_snippets/docs_snippets/intro_tutorial/basics/connecting_ops/
+cd examples/docs_snippets/docs_snippets/intro_tutorial/basics/connecting_ops/
```

Comment:
Since the instructions now tell you to `cd dagster` on line 17, i've updated the path accordingly